shared/ci/tasks/prep-release-src.sh

```diff
@@ -39,7 +39,7 @@ echo ""
 if [[ $(cat version/version) == "0.0.0" ]]; then
   echo "0.1.0" > version/version
 # Figure out proper version to release
-elif [[ $(cat artifacts/gh-release-notes.md | grep breaking) != '' ]] || [[ $(cat artifacts/gh-release-notes.md | grep feature) != '' ]]; then
+elif [[ $(cat artifacts/gh-release-notes.md | grep Breaking) != '' ]] || [[ $(cat artifacts/gh-release-notes.md | grep Feature) != '' ]]; then
```

Comment:
please use the case insensitive flag in grep, also, in regards to the logic not sure why but breaking changes must be major changes. @openoms do your remember why was implemented in this way?

Reply:
Ok, fixed in the other repo. Let's continue the discussion over there, that's easier to decide.
https://github.com/blinkbitcoin/concourse-shared/pull/7